devel-tools/update-changes

```diff
@@ -606,6 +606,11 @@ fi
 
 echo
 echo Type Enter to edit new $file_changes, or CTRL-C to abort without any modifications.
+if ! command -v gawk; then
```

Comment:
I cannot say how reasonable it is to expect `gawk`, but if we did this change we should also remove the redundant check in `default_format_msg`.

Reply:
Let me close this PR. This was a knee-jerk reaction after finding an optional dependency changes the formatting of the output silently. And yeah, either we should fully require it or not change anything.